manifests/export/pkcs12.pp

```diff
@@ -51,9 +52,10 @@
       $pass_opt,
       $passout_opt,
     ]
+    $_cmd = inline_template('<%= @cmd.join(" ") %>')
```

Comment:
In modern Puppet you can pass an array. Would it make sense to start using that? You'd need to modify `$cmd` a bit, so perhaps it's out of scope for this PR.

Also, Puppet itself can join without resorting to ERB templates:
```suggestion
$_cmd = $cmd.join(" ")
```

Reply:
done and amended